sensors/nxt_i2c_sensor_core.c

```diff
@@ -377,6 +399,18 @@ static int nxt_i2c_sensor_detect(struct i2c_client *client,
 #endif
 
 	/*
+	 * NXT Temperature sensor detection is handled here.
```

Comment:
Please add a sentence that explains the reason we are doing this. Technically, this driver is only for I2C devices that provide vendor and product id as per LEGO's unofficial spec. But we are making an exception since this is a LEGO brand sensor.